app/src/main/java/ca/chronofit/chrono/circuit/CircuitTimerActivity.kt

```diff
@@ -424,4 +419,12 @@ class CircuitTimerActivity : BaseActivity() {
             0
         }
     }
+
+    private fun getSoundFile(name: String): Int {
```

Comment:
Why move this function to here? Resource ID should be unique to the whole app, so calling R.raw.long_whistle from util should be the same as calling it here.